pkg/api/query/v1.go

```diff
@@ -1186,6 +1194,7 @@ func (qapi *QueryAPI) series(r *http.Request) (interface{}, []error, *api.ApiErr
 
 	q, err := qapi.queryableCreate(
 		enableDedup,
+		qapi.deduplicationFunc,
```

Comment:
also i don't think you need to add this to `queryableCreate` since  `queryableCreate query.QueryableCreator` is one of the params, all you need is to pass this arg to `NewQueryableCreator` ?

Reply:
I put it together with dynamically set params like the store selector and deduplication toggle as this one seemed to be fitting there more
But your are right, it's probably better to make the change less intrusive if this params is not exposed now.
I'll give it a bit of time and if no one else comment then I move it to querableCreator